sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLCommandTestUtils.scala

```diff
@@ -91,4 +91,20 @@ trait DDLCommandTestUtils extends SQLTestUtils {
   }
 
   protected def checkLocation(t: String, spec: TablePartitionSpec, expected: String): Unit
+
+  // Getting the total table size in the filesystem in bytes
+  def getTableSize(tableName: String): Int = {
```

Comment:
This PR shares the function with https://github.com/apache/spark/pull/31131 . As soon as one of them is merged, I will rebase.